README.md

```diff
@@ -279,6 +293,16 @@ This concludes my humble attempt to make a somewhat-useful “map” for an aspi
 
 > P.S. — This post is part of a open, collaborative effort to build an online reference, the Open Guide to Practical AI, which we’ll release in draft form soon. See* *[this popular guide](https://github.com/open-guides/og-aws)* *for an example of what’s coming. If you’d like to get updates on or help with with this effort, sign up* *[here](https://upscri.be/d29cfe/).
 
-Special thanks to* *[Joshua Levy](https://twitter.com/ojoshe),* *[Leo Polovets](https://twitter.com/lpolovets)* *and* *[Abhishek Das](https://www.linkedin.com/in/abhishek-das-3280053/)* *for reading drafts of this and their invaluable feedback!
+## Credits
+
+Original Author: [**Max Grigorev**](https://twitter.com/forwidur)
+
+Editorial help and feedback:
+[**Joshua Levy**](https://twitter.com/ojoshe),
+[**Leo Polovets**](https://twitter.com/lpolovets), and [**Abhishek Das**](https://www.linkedin.com/in/abhishek-das-3280053/)
+
+## License
```

Comment:
D: This could go into a separate LICENSE file. Not a strong opinion, but that seems to be the pattern usually.